### ISSUE TITLE
Anycubic *.photon files support

### ISSUE THREAD

**Issue by DenisNP** [alexrj/Slic3r#4617]
**Is your feature request related to a problem? Please describe.**
Slic3r is the only free software which can be used to slice DLP layers with _infill_. So, one of the best budget DLP printers is **Anycubic Photon**, which works with its own proprietary `.photon` format, not with USB-host or something similar.

**Describe the solution you'd like**
Add `Export to .photon` feature in DLP Projector window.

**Describe alternatives you've considered**
1. Buy something expensive like FormWare 3D Slicer.
2. Use infill-like modifications directly with the mesh in Meshmixer (Hollow + Pattern + Boolean operations)

**Comment by supermerill** [comment]
Did you try https://github.com/Photonsters/PhotonFileEditor?

**Comment by DenisNP** [comment]
@supermerill Yes. How can this help me? It doesn't support import from SVG, and Slic3r doesn't support export to PNG.

**Comment by lordofhyphens** [comment]
You could render the SVG to PNG with a tool like imagemagick.

We would need references for the format to even start

**Comment by foreachthing** [comment]
Or use Inkscape with this: https://inkscape.org/de/~Johannski/%E2%98%85inkscape-batch-convert-svgpdfeps-to-epspdfpng

**Comment by foreachthing** [comment]
@DenisNP You can make your own post-processing script from that batch-converter. Then, after exporting gcode (in Slic3r) it will convert all the svgs to pngs - done!

**Comment by DenisNP** [comment]
@foreachthing maybe I'll write `Slic3r SVG > .photon` converter someday. Haven't enough time for this right now :(

**Comment by DenisNP** [comment]
So [this is my temporary solution](https://github.com/DenisNP/Slic3rSVG2Photon)
